### ISSUE TITLE
Update and organize near.dev with open source examples from near-apps and awesomenear

### ISSUE THREAD

**Issue by thisisjoshford** [near/devrel#305]

**Comment by BenKurrek** [comment]
Creating a [list of open-source NEAR projects](https://docs.google.com/document/d/1Jx_Dz53bfK-QkeP2oGRzU51I9FqeIe_NNuoALwIrZQA/edit?usp=sharing). Will organize them into categories based on scale, and type of project.

**Comment by doriancrutcher** [comment]
For Near-apps I worry about option overload for newcomers to near since we aren't advertising create-near-app as much. Perhaps we should create an example pipeline for new devs of sorts to give them a starting place. And then advanced near devs can pick and choose the examples that fit their needs best?

**Comment by BenKurrek** [comment]
Agreed. I think Josh is working on a complete example for beginners that will go through and help users of all skill levels create a cool project from start to finish. It has several versions with increasing complexity and there is no prior knowledge required. I think if we advertise this and then at the end, advertise the near-apps for those keen developers, that would be a good compromise?
